tsconfig.json

```diff
@@ -16,5 +16,5 @@
     "useUnknownInCatchVariables": false
   },
   "include": ["src/**/*.ts"],
-  "exclude": ["node_modules", "dist", "fixtures", "src/templates", "pepr-test-module"]
```

Comment:
We can remove these because of the contents of `files` in `package.json`.

Reply:
pepr-test-module comes in handy in my local testing, but that's not a blocker for this PR, we can always bring it back when necessary

Reply:
`pepr-test-module` is usually created for integration tests and lives outside of `src/`, so it won't be picked up during build.